### ISSUE TITLE
Detected?

### ISSUE THREAD

**Issue by vahidx4r4x** [trojan-gfw/trojan#659]
Hi.
Thanks for your great project.
Im in a chimese telegram channel and i asked them that which protocol is the best and undetected. They said that Trojan is being blocked! But everywhere I searched for the answer I saw that people recommend using Trojan over VLESS and VMESS etc.
Is it true? If it is, whats the best protocol to use?
Does combining protocols with WS help?

Please help me.
Thanks

**Comment by yyuurrii34** [comment]
since early Nov of 2022,many protocols have reported being blocked issue,it is said that using utls will help ease these cases according to their discussion https://github.com/XTLS/Xray-core/issues/1253

**Comment by Leetroch** [comment]
Yeah. The Trojan is being detected, my trojan server has been blocked over twice. Although I have changed my IP of the server and blocked ports except 443 and 80, it just works for a few days. I guess if you want to make it works, you will need a new agent software with CDN or UDP.
